lib/spectrum/response/specialists.rb

```diff
@@ -87,10 +87,9 @@ def client
       end
 
       def fetch_records(query)
-        params = focus.first.solr_params.merge(
+        params = focus.first.solr_params.merge(query).merge(
           q: query[:q],
           fq: query[:fq],
-          qq: '"' + RSolr.solr_escape(query[:q]) + '"',
           rows: rows.first,
           fl: fields.first,
           df: query[:df] || 'allfields'
```

Comment:
Are we sure this should ever happen (that :df and :qf aren't set)? I'm happy with the workaround, but I wish we knew what exactly we're working around.

Reply:
It's possible that I could and should do a little bit of refining - for instance, since I'm merging in the query itself now, and the query should have a predictable shape, _probably_ all we need to set is `rows` and `fl`?